packages/client/test/net/protocol/ethprotocol.spec.ts

```diff
@@ -70,7 +70,7 @@ describe('[EthProtocol]', () => {
       'encode status',
     )
     const status = p.decodeStatus({
-      chainId: [0x01],
+      chainId: Uint8Array.from([0x01]),
```

Comment:
Why is this change now necessary?

Reply:
the function accepted arrays before. Now it is limited to u8a

Reply:
Furthermore, this property is supposed to be a Uint8Array if you look at the types.